DigitalLearningSolutions.Web/ViewModels/SuperAdmin/Faqs/SearchableFaqModel.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Faqs
```

Comment:
This looks to be a duplicate of `SearchableFaq` used in support. 
Can we drop one and share the model across both?

Reply:
had a go but it didnt seem to live them not being in the same namespace?

Reply:
In terms of sharing I would look for a common place for the class to live like `Web.ViewModels.Shared.Faqs`.
A simple alternative could be to inherit instead. If you want to give that a try. 
This class here would end up looking like this instead:
```
namespace DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Faqs
{
    using DigitalLearningSolutions.Data.Models.Support;
    using BaseSearchableFaqModel = Support.Faqs.SearchableFaq;

    public class SearchableFaqModel : BaseSearchableFaqModel
    {
        public SearchableFaqModel() : base() { }

        public SearchableFaqModel(Faq model) : base(model) { }
    }
}
```

Reply:
ahh cool thanks I've put them in a shared directory

Reply:
Nice one